Source/Urho3D/Urho2D/TmxFile2D.cpp

```diff
@@ -527,6 +525,17 @@ bool TmxFile2D::EndLoad()
     return true;
 }
 
+bool TmxFile2D::IsCorrectVersion(const XMLElement& element) const
+{
+    const String version{ element.GetAttribute("version") };
+
+    if (version.StartsWith("1."))
+        return true;
+
+    URHO3D_LOGERROR("Invalid TMX version");
```

Comment:
I would suggest printing the invalid version to the log as well, e.g. 
`URHO3D_LOGERROR("Invalid TMX version: " + version);

Reply:
That's good point.
I'll merge functional part of this PR later, I can add this.

I have no intention to merge this PR as is, because (1) it is political PR aimed to re-style existing codebase without any prior discussion and/or request, (2) it's about 20 times bigger than it needs to be and (3) it actually goes _against_ Urho coding conventions and, surprisingly, some items of C++ Code Guidelines -- only for sake of changing more lines of code.